File: SMB_Int2022_pr4_s26752_MateuszBieliński/ShoppingList/Model/Product.swift
//
//  Product.swift
//  ShoppingList
//
//  Created by Mateusz Bieliński on 06/12/2022.
//

import Foundation
import FirebaseFirestoreSwift

struct Product: Identifiable, Codable {
    @DocumentID var id: String?
    var name: String
    var price: Double
    var amount: Int
    var isBought: Bool
}

File: SMB_int2022_pr2_s26752_MateuszBieliński/ShoppingList/java/com/example/shoppinglist/Product.kt
package com.example.shoppinglist

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "product_table")
data class Product(@PrimaryKey(autoGenerate = true) var id: Long = 0, var name: String, var amount: Int, var price: Double, var isBought: Boolean){
}